### ISSUE TITLE
Popup and tooltip - aria-describedby value should be controlled

### ISSUE THREAD

**Issue by fleish80** [ng-bootstrap/ng-bootstrap#2472]
### Bug description:

Aria-describedby value is updated automatically with id value which is found on popup/tooltip window root element. But there are some cases which popup/tooltip windows contain too much information, and then it's necessary to put some internal id in aria-describedby.
Excepted behaviour is to allow control with aria-describedby value.

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular:
6.0.0
ng-bootstrap:
2.1.1
Bootstrap:
4.0.0

**Comment by giampierobono** [comment]
I think information provided in the website should be same for "normal" and disabled users. 

Since the semantic meaning of tooltips is to clarify or help user in using the content that they hover over, not add additional content, I would suggest to avoid removing part of this text. 

This could be done for popover, having a different semantic meaning (even if following the principle to provide same information for all users, I would suggest to use carefully this feature). 

What do you think?

**Comment by giampierobono** [comment]
Let me know :) in case i have PR ready for both tooltips and popover

**Comment by fleish80** [comment]
I agree with you that this principle should be used very carefully, so I suggest that maybe by default to leave it as it now, but I still think that you should reveal us three inputs, aria-label, aria-labelledby and aria-describedby and let the developer to decide when to use each one of them and of course to override the default behaviour if aria-describedby was defined.
